ambry-network/src/main/java/com.github.ambry.network/NetworkMetrics.java

```diff
@@ -30,23 +31,30 @@
 public class NetworkMetrics {
   private final MetricRegistry registry;
 
-  // Selector metrics
+  // Rates and Counters
+  // Selector
   public final Counter sendInFlight;
   public final Counter selectorConnectionClosed;
   public final Counter selectorConnectionCreated;
   public final Counter selectorSelectRate;
-  public final Histogram selectorSelectTime;
   public final Counter selectorIORate;
-  public final Histogram selectorIOTime;
-  public final Counter selectorNioCloseErrorCount;
-  public final Counter selectorDisconnectedErrorCount;
-  public final Counter selectorIOErrorCount;
-  public final Counter selectorKeyOperationErrorCount;
-  public final Counter selectorCloseKeyErrorCount;
-  public final Counter selectorCloseSocketErrorCount;
-  public Gauge<Long> selectorActiveConnections;
-  public final Map<String, SelectorNodeMetric> selectorNodeMetricMap;
+  // SSL metrics
+  public final Counter sslFactoryInitializationCount;
+  public final Counter sslTransmissionInitializationCount;
+  public final Counter sslHandshakeCount;
+  // the count of renegotiation after initial handshake done
+  public final Counter sslRenegotiationCount;
+  // NetworkClient metrics
+  // number of times connection checkout have been tried before succeeding
+  public final Meter connectionCheckOutAttemptsCount;
+  // number of times connection checkout have been tried before timing out
+  public final Meter connectionCheckOutFailureAttemptsCount;
+  public final Meter connectionsDisconnectedCount;
```

Comment:
Already we have metrics in similar names. So going with the same. Do check NettyMetrics for connectionsDisconnectedCount